.github/workflows/release.yml

```diff
@@ -29,6 +29,7 @@ jobs:
         run: |
           echo "::set-env name=TAG::${GITHUB_REF#refs/*/v}"
           echo "::set-env name=RELEASE_TAG::${TAG}"
+          echo "RELEASE_TAG ${TAG}"
```

Comment:
Is this for debug purpose? should it be removed?

Reply:
It will help in debugging also. With this we can know the tag with which the image is being built. if buildx fails in some intermediate step, it does not error out with the image name it was trying to build. Therefore included the env.

I think we should keep it.